### ISSUE TITLE
Can't see prisons/imported on regions/province plots

### ISSUE THREAD

**Issue by djay** [djay/covidthailand#196]
Makes it hard to explain the cause of overall trend without including numbers.
Previously we included both.
![image](https://user-images.githubusercontent.com/41700/142791891-5a7862e3-0f16-4d36-81ab-6e10feb260ed.png)

Maybe not separate prisons out as a seperate region on the /100k plots since the small population (300k) might make it seem like a greater contribution than it is. I'm not sure we have a number for total people crossing the border to give imported/100k?
but it could be in the table and on the stacked plot?
![image](https://user-images.githubusercontent.com/41700/142792223-78c4ae12-77df-4b32-ad98-549f67c0e9b4.png)

Not sure if prison covid deaths are still separated out from the regional data or not.

**Comment by modiholodri** [comment]
@djay Maybe there should be a Others Region to include the rest (prison/imported/...). The Other region might grow also once the tourists start coming again...

**Comment by djay** [comment]
@modiholodri maybe. not sure on combining them since they are very different sources and currently prisons is much higher than imported. On the other hand they do have one thing in common that they are both populations outside the normal community with not much chance of community spread. non-community region?

however we still can't make it a region in the 100k plots since we don't know the population of visitors I think... There is viisitor data they are releasign daily and everyone stays one night so the population of quarantine is just the daily number. But I think imported cases can come from people crossing the border who don't have to quarantine...

**Comment by modiholodri** [comment]
@djay maybe just add the others to the stacked plots. No need to show them per 100k.

**Comment by djay** [comment]
yeah could do. but would also be good to have them separately in the table as well esp with the arrow indicator. Not sure if we can squeeze them in under central?
